systest/21million/common/run_queries.go

```diff
@@ -18,53 +18,58 @@ package common
 
 import (
 	"context"
+	"fmt"
 	"io"
+	"log"
 	"os"
+	"os/exec"
 	"path/filepath"
 	"runtime"
 	"strings"
-	"testing"
 	"time"
 
+	"github.com/pkg/errors"
+
 	"github.com/dgraph-io/dgraph/chunker"
-	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/dgraph-io/dgraph/dgraphtest"
 )
 
-// JSON output can be hundreds of lines and diffs can scroll off the terminal before you
-// can look at them. This option allows saving the JSON to a specified directory instead
-// for easier reviewing after the test completes.
-//var savedir = flag.String("savedir", "",
-//	"directory to save json from test failures in")
-//var quiet = flag.Bool("quiet", false,
-//	"just output whether json differs, not a diff")
-
-func TestQueriesFor21Million(t *testing.T) {
-	_, thisFile, _, _ := runtime.Caller(0)
-	queryDir := filepath.Join(filepath.Dir(thisFile), "../queries")
+var datafiles = map[string]string{
+	"1million-noindex.schema": "https://github.com/dgraph-io/benchmarks/blob/master/data/1million-noindex.schema?raw=true",
+	"1million.schema":         "https://github.com/dgraph-io/benchmarks/blob/master/data/1million.schema?raw=true",
+	"1million.rdf.gz":         "https://github.com/dgraph-io/benchmarks/blob/master/data/1million.rdf.gz?raw=true",
+	"21million.schema":        "https://github.com/dgraph-io/benchmarks/blob/master/data/21million.schema?raw=true",
+	"21million.rdf.gz":        "https://github.com/dgraph-io/benchmarks/blob/master/data/21million.rdf.gz?raw=true",
+}
 
+func QueriesFor21Million(dc dgraphtest.Cluster) error {
```

Comment:
QueriesFor21Million -> RunTestQueriesFor21Million
and I think we should pass t object to this function so that we could do `t.Run` inside.